ops/docker-compose-omgx.yml

```diff
@@ -126,7 +126,7 @@ services:
       L2_NODE_WEB3_URL: http://l2geth:8545
       URL: http://deployer:8081/addresses.json
       SEQUENCER_PRIVATE_KEY: "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
-      PROPOSER_PRIVATE_KEY: "0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a"
```

Comment:
FYI - I think deleting the proposer key has an unexpected side effect, which is that the proposer then just uses the `SEQUENCER_PRIVATE_KEY`, which can lead to the nonce errors

Reply:
could be! but this is not my doing... this file is generated from the original `ops/docker-compose.yml`